src/main/java/org/axion/axion_api_gateway/AxionApiGatewayApplication.java

```diff
@@ -12,9 +12,11 @@ public class AxionApiGatewayApplication {
     private static final Logger log = LoggerFactory.getLogger(AxionApiGatewayApplication.class);
 
     private static AppConfig appConfig;
+    private static UserSession userSession;
 
     public static void main(String[] args) {
         appConfig = new AppConfig();
+        userSession = new UserSession(appConfig);
```

Comment:
Moved this from local variable to class level field, so I can user this in two methods.